Misc/NEWS.d/next/Build/2017-11-02-23-10-08.bpo-31926.tbb5xH.rst

```diff
@@ -0,0 +1,4 @@
+rgument Clinic failing to generate some ``#ifndef ... #define`` blocks for
```

Comment:
:scissors:

Reply:
Done.

Can this go in?

Reply:
Sorry for been too terse and ambiguous. I meant that the NEWS entry were cut (missed starting "A" and likely "Fixed"), not that it should be shortened. But in any case the new wording looks better to me!